packages/logs/lib/otlp/otlp.ts

```diff
@@ -53,10 +52,17 @@ async function updateRoutes(getRoutes: () => Promise<RouteConfig[]>) {
 }
 
 export const otlp = {
+    running: false,
     register: once(async (getRoutes: () => Promise<RouteConfig[]>) => {
-        await updateRoutes(getRoutes);
-        setInterval(async () => await updateRoutes(getRoutes), 10000);
+        otlp.running = true;
+        while (otlp.running) {
+            await updateRoutes(getRoutes);
+            await new Promise((resolve) => setTimeout(resolve, 15000));
```

Comment:
nb: you can use `node:timers` for this natively

Reply:
always forget about it, even though that what I used in the scheduler package. Trying to deploy asap though so leaving it